hasura.planx.uk/migrations/1709208323190_run_sql_migration/up.sql

```diff
@@ -0,0 +1,54 @@
+DROP VIEW public.analytics_summary;
+
+CREATE
+OR REPLACE VIEW public.analytics_summary AS
+select
+	a.id as analytics_id,
+	al.id as analytics_log_id,
+	f.slug as service_slug,
+	t.slug as team_slug,
+	a.type as analytics_type,
+	a.created_at as analytics_created_at,
+	(user_agent -> 'os' ->> 'name') :: text AS operating_system,
+	(user_agent -> 'browser' ->> 'name') :: text AS browser,
+	(user_agent -> 'platform' ->> 'type') :: text AS platform,
+	referrer,
+	flow_direction,
+	metadata ->> 'change' as change_metadata,
+	metadata ->> 'back' as back_metadata,
+	metadata ->> 'selectedUrls' as selected_urls,
+	metadata ->> 'flag' as result_flag,
+	metadata -> 'flagSet' as result_flagset,
+	metadata -> 'displayText' ->> 'heading' as result_heading,
+	metadata -> 'result' ->> 'description' as result_description,
```

Comment:
I don't think the `result` key exists in the metadata:

```JSON
{
      "metadata": {
        "back": {
          "id": "qwMn0D7gnF",
          "type": "Notice",
          "title": "Tree consent"
        },
        "flag": {
          "text": "Permission needed",
          "color": "#000000",
          "value": "PLANNING_PERMISSION_REQUIRED",
          "bgColor": "#A8A8A8",
          "category": "Planning permission",
          "description": "It looks like the proposed changes may require planning permission."
        },
        "flagSet": "Planning permission",
        "displayText": {
          "heading": "It looks like this project needs planning permission",
          "description": "Based on the information you have provided, this project needs planning permission. Continue to find out how to apply."
        },
        "isAutoAnswered": false
      }
    }
```

 Is this maybe supposed to be `displayText` to access the nested description?

Reply:
Yep - good catch sorry - copy paste issue!